DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs

```diff
@@ -89,6 +91,8 @@ int sectionId
                 tutorialInformation.DiagnosticAssessmentCompletionThreshold,
                 tutorialInformation.TutorialsCompletionThreshold
             );
+            TutorialStartButtonColour = tutorialInformation.Status == "Complete" ? "nhsuk-button--secondary" : "";

```

Comment:
I think `TutorialStartButtonStyling` or even `TutorialStartButtonAdditionalStyling` might be a more descriptive name? 🙂

Reply:
Thanks! I was fighting with a variable name for that and couldn't for the life of me think of one aha